### DISCUSSION TITLE
Filter i3 Workspaces

### DISCUSSION THREAD

**Discussion by anamyk** [polybar/polybar#3083] [Q&A]
Hi all,

is there are possibility to filter i3 workspaces in polybar, so that one can specify different behaviours for different workspace groups?

E.g.
for workspaces 1-10 do this
for workspaces 11-20 do that

Any ideas and help appreciated!

**Comment by patrick96** [comment]
This is not directly supported. But you can maybe achieve something like this by using `ws-icon` to exclude workspace names by mapping them to the empty string effectively making them disappear

```dosini
ws-icon-0 = Workspace Name;
...
label-focused = %icon%
label-unfocused = %icon%
label-visible = %icon%
label-urgent = %icon%
```

https://github.com/polybar/polybar/wiki/Module:-i3
